Bug in get last fix (#19325)
demisto/content: Packs/GitLab/ReleaseNotes/1_0_1.md

#### Integrations
##### GitLab Event Collector

> - Improved implementation of the get last function to support a case with empty events.

Reviewer: ```suggestion
- Improved implementation of the **get_last_run** function, to support empty events.
```
